src/pages/404.js

```diff
@@ -1,5 +1,12 @@
-import React from 'react';
+import React from "react";
+import NavBar from "../components/organisms/NavBar/Navbar";
+import NotFoundPageTemplate from "../components/templates/NotFoundPageTemplate";
 
-const NotFound = () => <div>404!</div>;
+const NotFound = () => (
+  <div>
+    <NavBar />
```

Comment:
Put the navbar into the template.. pages are only used to render templates + handling logic